### PR TITLE
Add SDR UK to jobs page

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jacobpgn , @hmac : please review.

### CODE REVIEW

**Comment by hmac** [comment]
Seems legit, but I have no context on this thing. Defer to @jacobpgn

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        <p className='para'>
+          The sales team at GoCardless combines deep product expertise with a
+          commitment to helping new and existing businesses succeed across a
+          range of industries. As a Sales Development Rep, you’ll be the first port of call for many of our prospective customers.
+          You’ll mix in-depth knowledge of the product and wider payments industry with a consultative sales approach to
+          help fit the product to the customer’s specific requirements. This is a high impact role and you’ll have the unique
+          opportunity to work closely with Marketing, Product and Ops and shape the way our product is used.
+        </p>
+        <p className='para'>
+          Training for the position will involve a <a href='https://gocardless.com/blog/how-we-train-salespeople/' target='_blank'>  4 week structured program </a>
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: triple-space before 4!

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            high-growth environment.
+          </li>
+          <li>
+            You have superb verbal and written communication skills.
+          </li>
+        </ul>
+        <p className='para'>
+          Our team comes from a variety of different background and includes entrepreneurs, consultants and fresh graduates.
+          To help you, we've written a <a href='https://gocardless.com/blog/the-account-executive-interview-process/' target='_blank'>
+          blog post </a> which takes you through our interview process and gives you some tips on what to expect. We've also
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: double-space

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          identified <a href='https://gocardless.com/blog/how-we-train-salespeople/' target='_blank'> five characteristics </a>
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: spaces everywhere

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+          that are vital to be a successful member of our sales team.
+        </p>
+        <h4 className='u-text-xs u-margin-Bxs u-margin-Tm'>The perks</h4>
+        <ul className='list u-margin-Tm'>
+          <li>
+            Competitive salary, commission and options package.
+          </li>
+          <li>
+            Macbook Air and 27" Thunderbolt display.
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: `MacBook Air` and no more Thunderbolts 🍏

File: app/pages/about/jobs/positions/sales-development-representative-inbound.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+          </li>
+          <li>
+            Bi-weekly and monthly all team lunches and evenings out.
+          </li>
+          <li>
+            Stocked kitchen and unlimited (great) coffee!
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: Unmetered. There's a limit. It's when you start seeing sounds.

Comment (gabrielamatias) [inline app/pages/about/jobs/positions/sales-development-representative-inbound.js]: I'm |-| this close to adding that as a joke 😛

**Pull request merged**